docs/src/content/docs/it/guides/sidebar.mdx

```diff
@@ -260,12 +268,30 @@ La configurazione sopra genera la seguente barra laterale:
 <SidebarPreview
 	config={[
 		{
-			label: 'Guide',
+			label: 'Guides',
 			items: [
 				{
-					label: 'Componenti',
+					label: 'Components',
 					link: '/guides/components/',
-					badge: { text: 'Nuovo', variant: 'default' },
+					badge: { text: 'New', variant: 'default' },
+				},
+			],
+		},
+		{
+			label: 'Reference',
+			badge: { text: 'Deprecated', variant: 'default' },
+			items: [
+				{
+					label: 'Configuration Reference',
+					link: '/reference/configuration/',
+				},
+				{
+					label: 'Frontmatter Reference',
+					link: '/reference/frontmatter/',
+				},
+				{
+					label: 'Overrides Reference',
+					link: '/reference/overrides/',
```

Comment:
We usually translate labels in this case and link to the proper language so the user does change language if they ever test following the link.